client/src/components/screens/PatientVariant/index.jsx

```diff
@@ -1047,109 +1026,106 @@ class PatientVariantScreen extends React.Component {
     const reportAvailable = this.isReportAvailable();
 
     return (
-      <Content>
-        <Card className="entity" bordered={false}>
-          <VariantNavigation
-            key="variant-navigation"
-            className="variant-navigation"
-            schema={schema}
-            patient={patient}
-            queries={draftQueries}
+      <Card className="entity" bordered={false}>
```

Comment:
Here Github doesn't seems to understand to formatiting + removing the `content` component. I copy pasted the "content" of `content` 🙄 to make sure it stays the the